README.md

```diff
@@ -43,6 +44,7 @@ Refer to the table below for a detailed view of final models materialized by def
 # 🎯 How do I use the dbt package?
 ## Step 1: Pre-Requisites
 - **Connector**: Have all relevant Fivetran app platform connectors syncing data into your warehouse. This package currently supports:
```

Comment:
I just realized this reads a bit strange and makes it seem like you need **all** the platforms in order to run the package. Can you update this to provide the context that you just need at least one (but best results for multiple) platform(s).

Reply:
Additionally, in step 2 of the instructions we still reference the old range. Since this is a breaking change we will want to reference the new range
```yaml
packages:
  - package: fivetran/ad_reporting
    version: [">=1.2.0", "<1.3.0"]
```

Reply:
Done.